app/javascript/gobierto_visualizations/webapp/lib/utils.js

```diff
@@ -1,6 +1,9 @@
 import { csv } from "d3-fetch";
+import { scaleOrdinal } from 'd3-scale';
 import axios from "axios";
 
+const d3 = { scaleOrdinal }
```

Comment:
Get rid of `d3` namespace. Look at `csv` function

Reply:
Done!